crates/ruff/src/rules/flake8_pytest_style/rules/parametrize.rs

```diff
@@ -166,6 +169,58 @@ impl Violation for PytestParametrizeValuesWrongType {
     }
 }
 
+/// ## What it does
+/// Checks for duplicate test cases in `pytest.mark.parametrize`.
+///
+/// ## Why is this bad?
+/// Duplicate test cases are redundant and should be removed.
+///
+/// ## Example
+/// ```python
+/// import pytest
+///
+///
+/// @pytest.mark.parametrize(
+///     ("param1", "param2"),
+///     [
+///         (1, 2),
+///         (1, 2),
+///     ],
+/// )
+/// def test_foo(param1, param2):
+///     ...
+/// ```
+///
+/// Use instead:
+/// ```python
+/// import pytest
+///
+///
+/// @pytest.mark.parametrize(
+///     ("param1", "param2"),
+///     [
+///         (1, 2),
+///     ],
+/// )
+/// def test_foo(param1, param2):
+///     ...
+/// ```
+///
+/// ## References
+/// - [`pytest` documentation: How to parametrize fixtures and test functions](https://docs.pytest.org/en/latest/how-to/parametrize.html#pytest-mark-parametrize)
+#[violation]
+pub struct PytestDuplicateParametrizeTestCases {
+    pub indices: (usize, usize),
+}
+
+impl Violation for PytestDuplicateParametrizeTestCases {
+    #[derive_message_formats]
+    fn message(&self) -> String {
+        let PytestDuplicateParametrizeTestCases { indices } = self;
+        format!("Found duplicate test cases {indices:?} in `@pytest.mark.parametrize`")
```

Comment:
should we say "at indices ..."? I wasn't sure what the message meant at first.

Reply:
Can we raise PT014 against items that should be removed?

```
[1, 1, 2]
    ^ PT014 ...
```

Reply:
I think it makes sense to use the range of the duplicated item

Reply:
```
# Example

[100, 100, 100, 200]
      ^^^  ^^^
```

If we have multiple duplicated items, we underline them?

Reply:
I don't think we support highlighting multiple ranges. In that case, they'd each need to be a new violation which seems okay.

Reply:
I think in the duplicate value rule, we just highlight the _second_ value (i.e., the one that is a duplicate). That seems reasonable to me. (We could also mention the index of which it's a duplicate in the message.)

Reply:
```
# Example

[100, 100, 100, 200]
      ^^^ PT014: duplicate of item at {0}
```

so it should look like this?